### DISCUSSION TITLE
Change of company name - repository settings update required

### DISCUSSION THREAD

**Discussion by DotNetDublin** [BlazingSaddlesSolutions/TimeKeeper#54] [General]
I was reading the other day that in some respects Blazing Saddles hasn't aged so well so without debating that I've changed the company name to simply Blazing Solutions.

One impact the above change has is in updating the repository URL from https://github.com/BlazingSaddlesSolutions/TimeKeeper to https://github.com/BlazingSolutions/TimeKeeper

I think you have two options in terms of updating your local settings.

One would be to start from scratch, delete what you have and clone the repository again.

The second is to follow the below simple steps in Visual Studio.

![step1](https://user-images.githubusercontent.com/52583820/119119410-14c4de00-ba23-11eb-8c00-21cc12cbb797.jpg)

![step2](https://user-images.githubusercontent.com/52583820/119119408-142c4780-ba23-11eb-9f05-0a6577b55368.jpg)

![step3](https://user-images.githubusercontent.com/52583820/119119405-142c4780-ba23-11eb-8073-76cfff1b7fbf.jpg)

![step4](https://user-images.githubusercontent.com/52583820/119119403-1393b100-ba23-11eb-920d-866889ff2911.jpg)

**Comment by harperjohn** [comment]
did a clean clone. Set my database connection string. And all is well!  (I like the new name! I like the old name too, but I guess the new name is "safer"... :) )
